sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1134,13 +1136,13 @@ pi_result piextUSMEnqueueMemAdvise(
   int          advice,
   pi_event *   event);
 
-/// API to query information about USM pointers including type and
-/// device allocated against
-/// Examples include:
-///   PI_MEM_ALLOC_TYPE
-///   PI_MEM_ALLOC_BASE_PTR
-///   PI_MEM_ALLOC_SIZE
-///   PI_MEM_ALLOC_DEVICE
+/// API to query information about USM allocated pointers
+/// Valid Queries:
+///   PI_MEM_ALLOC_TYPE returns host/device/shared
+///   PI_MEM_ALLOC_BASE_PTR returns the base ptr of an allocation if
+///                         the queried pointer fell inside an allocation
+///   PI_MEM_ALLOC_SIZE returns how big the allocation the queried pointer is
```

Comment:
Please specify the PI type returned here (pi_uin32 or pi_uint64), which is vital for correct implementations.

Reply:
void *